crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -150,6 +157,48 @@ async fn invalid_nonce_failure_causes_tx_resubmission_under_different_nonce() {
     .expect("valid nonce guard failed");
 }
 
+#[tokio::test]
+async fn test_single_tx_integrity() {
```

Comment:
don't prefix tests with `test_` as it's redundant. Rather give it a name describing what you want to observe. `async fn composer_forwards_transaction_unchanged` or something like that.